src/containers/audio-trimmer.jsx

```diff
@@ -52,7 +52,7 @@ class AudioTrimmer extends React.Component {
     render () {
         return (
             <AudioTrimmerComponent
-                containerRef={el => (this.containerElement = el)}
+                containerRef={el => (this.containerElement = el)} // eslint-disable-line react/jsx-no-bind
```

Comment:
Well my thinking was basically that this is identical to the way you would do it with a regular ref (which eslint wouldn't complain about, because the rule doesn't apply for refs).